polymetis/tests/scripts/4_continuous_grasping_hw.py

```diff
@@ -3,27 +3,24 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 import sys
-from polymetis.utils.continuous_grasper import ManipulatorSystem
-
 
-DEFAULT_MAX_ITERS = 3
+from polymetis.utils.continuous_grasper import ManipulatorSystem
 
+DEFAULT_MAX_ITERS = 3 
 
-def main(argv):
-    if len(argv) > 1:
-        try:
-            max_iters = int(argv[1])
-        except ValueError as exc:
-            print("Usage: python 5_continuous_grasping.py <max_iterations>")
-            return
-    else:
+def main(max_iterations, **kwargs):
+    try:
+        max_iters = int(max_iterations)
+    except ValueError as exc:
+        print("Malformed iteration count, using default")
         max_iters = DEFAULT_MAX_ITERS
-
-    # Initialize interfaces
-    robot = ManipulatorSystem()
+    robot = ManipulatorSystem(robot_kwargs=kwargs, gripper_kwargs=kwargs)
```

Comment:
I think passing `kwargs` to both robot_kwargs and gripper_kwargs can have odd effects (eg when gripper & robot ip_address are not the same, which can be common).

Reply:
This works as long as keys do not clash right ?

Reply:
Yes, but the keys may often clash (e.g. `ip_address` is shared but gripper should be connected to robodev, not NUC).